torchmetrics/functional/image/ssim.py

```diff
@@ -274,7 +274,7 @@ def structural_similarity_index_measure(
         >>> structural_similarity_index_measure(preds, target)
         tensor(0.9219)
     """
-    preds, target = _ssim_update(preds, target)
```

Comment:
I wonder if we shouldn't default to a single value (11) here and expand that to the necessary dimensions based on `preds.ndim` or `target.ndim`. Would be cleaner for 2d and 3d then :) Especially if this is a default variable

Reply:
That would be possible. A single StructuralSimilarityIndexMeasure object could then even handle 2d as well as 3 data.

Reply:
However, we would still need to allow the user to provide the sequence manually for anisotropic kernels I think :)

Reply:
Sure, that's still possible